main.yml

```diff
@@ -207,6 +207,7 @@ ccx:
   api:
     versions:
       - v1
+      - v2
```

Comment:
I've checked this in the app and looks like we support just one api version, the first one is listed in the table. So adding the `v2` before `v1` is the right way. I'll create a ticket in our JIRA to support multiple versions where we'd show multiple tags in the version column and users will be able to click on either of these values (the first one will be used as default when clicking on title in table)